src/containers/Dev/Snapshots.tsx

```diff
@@ -193,16 +216,24 @@ const Snapshots: React.FC = () => {
     const [value, setValue] = useState('')
     if (compareFrom !== '') return null
     return (
-      <Grid.Row key={`app_menu_new-snapshot`}>
-        <div>
-          <Input
-            size="mini"
-            onChange={(_, { value }) => setValue(value)}
-            placeholder="New Snapshot"
-          />
-          <Icon className="clickable" name="record" onClick={() => takeSnapshot(value)} />
-        </div>
-      </Grid.Row>
+      <>
+        <Table.Cell>
+          <div className="flex-row-start" style={{ alignItems: 'center' }}>
+            <Input
+              size="mini"
+              onChange={(_, { value }) => setValue(value)}
+              placeholder="New Snapshot"
```

Comment:
I guess we are skipping localisations for snapshot currently?

Reply:
For now, I guess. We should update them if/when we make this tool a bit more user-focused. But for now only devs should be going near it.